auth/standard_role_manager.cc

```diff
@@ -132,6 +134,25 @@ const resource_set& standard_role_manager::protected_resources() const {
     return resources;
 }
 
+static future<> add_options_column_if_missing(database& db, ::service::migration_manager& mm) noexcept {
+    try {
+        auto schema = db.find_schema(sstring(meta::AUTH_KS), sstring(meta::roles_table::name));
+        bytes options_name = to_bytes(meta::roles_table::options_col_name.data());
+        if (schema->get_column_definition(options_name)) {
+            return make_ready_future<>();
+        }
+        schema_builder b(schema);
+        b.with_column(options_name,
+                map_type_impl::get_instance(utf8_type, utf8_type, false),
+                column_kind::regular_column);
+        schema_ptr table = b.build();
+        return mm.announce_column_family_update(table, false, {}).handle_exception([] (const std::exception_ptr&) {});
+    } catch (...) {
```

Comment:
What happens if these run in parallel on two nodes?

I believe it should be fine. The new schema would be the same on both nodes, and receive the same digest.

@tgrabiec for verification.